.github/ISSUE_TEMPLATE/amend-existing-component.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Amend existing component
+about: Propose an improvement to an existing component
+---
+
+## Component to amend
+
+Add component name(s) here,
+
+## Visual
+
+Provide a screenshot or link to a prototype of your component amendment.
```

Comment:
Not sure you'd have that at this time, but I guess you can delete the section?

Reply:
Yeah, I quite like visuals, but of course, the issue author can remove it if it doesn't apply.